index.ts

```diff
@@ -114,6 +114,12 @@ const App = createApp({
   experimental: {
     earn: {
       showSafetyScoreOnPoolCard: true,
+    },
+    wallet: {
+      showActionsCarousel: true,
+    },
+    activity: {
+      hideActionsCarousel: true
```

Comment:
one of them is hide and one is show? 😢

Reply:
We want boolean defaults to be false and this is one way to accomplish that: [thread](https://github.com/divvixyz/divvi-mobile/pull/65#pullrequestreview-2599626222) for context.